add-on/panels/js/roomStore.js

```diff
@@ -544,7 +550,8 @@ loop.store = loop.store || {};
      */
     openRoom: function(actionData) {
       loop.requestMulti(
-        ["Rooms:Open", actionData.roomToken],
+        ["OpenURL", TOC_URL + actionData.roomToken],
+        ["LoadSidebar", actionData.roomToken],
```

Comment:
I think these should really be in `LoopRooms#open`. The reason is, that as we move to a new window, we're going to want to make sure that we only have one instance of a room open at any one time. The only place we can really do that is from the privileged code.

So I think it makes sense to keep using that function for now (I'm not expecting the one-instance code to be written yet).

I realise there's some XXX code for `LoopRooms#open`, maybe just comment out the old code within the function and add the XXX akita to tidy up all chat window associated code later (maybe file a bug at the same time?)